modules/uninstalling-olm-using-ansible.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * install_config/installing-operator-framework.adoc
+
+[id='uninstalling-olm-using-ansible_{context}']
+= Uninstalling Operator Lifecycle Manager using Ansible
+
+To uninstall the Technology Preview Operator Framework, you can use the included
```

Comment:
This sentence is a little confusing to me. Maybe omitting the included playbook? 
```suggestion
After installing your cluster, you can use this procedure with the {product-title} `openshift-ansible` installer to uninstall the Technology Preview Operator Framework.
```

Reply:
I'd probably say:
> After installing your cluster, you can uninstall the Technology Preview Operator Framework by using he {product-title} `openshift-ansible` installer.

It puts the action closer to the start of the sentences and avoids self-referential language.

Reply:
Addressed